composer.json

```diff
@@ -21,7 +21,8 @@
 
 	"require-dev": {
 		"satooshi/php-coveralls": "dev-master",
-		"phpunit/php-file-iterator": "1.3.3"
+		"phpunit/php-file-iterator": "1.3.3",
+		"phpunit/phpunit": "~4.0"
```

Comment:
Travis-CI uses PHPUnit 4.5.0 currently. But there are many devs out there, that have a local installation of 3.X. Also at the end of 2015 PHPUnit v5.0 should be released.

Having PHPUnit as require-dev lets you be sure, that:
1. Everybody can use the correct version of PHPUnit for testing.
2. The requirements for PHPUnit are checked by composer on installation.